indigo_api/models.py

```diff
@@ -945,23 +974,12 @@ class Meta:
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
-    last_submitted_by_user = models.ForeignKey(User, related_name='+', null=True, on_delete=models.SET_NULL)
-    last_submitted_at = models.DateTimeField(null=True)
-
-    last_cancelled_by_user = models.ForeignKey(User, related_name='+', null=True, on_delete=models.SET_NULL)
-    last_cancelled_at = models.DateTimeField(null=True)
-
-    last_reopened_by_user = models.ForeignKey(User, related_name='+', null=True, on_delete=models.SET_NULL)
-    last_reopened_at = models.DateTimeField(null=True)
-
-    last_unsubmitted_by_user = models.ForeignKey(User, related_name='+', null=True, on_delete=models.SET_NULL)
-    last_unsubmitted_at = models.DateTimeField(null=True)
-
-    last_closed_by_user = models.ForeignKey(User, related_name='+', null=True, on_delete=models.SET_NULL)
-    last_closed_at = models.DateTimeField(null=True)
-
     labels = models.ManyToManyField('TaskLabel', related_name='+')
 
+    @property
+    def place(self):
+        return self.locality if self.locality else self.country
+
     @property
     def place_code(self):
         return self.country.code + '-' + self.locality.code if self.locality else self.country.code
```

Comment:
This can now change to `self.place.place_code` which hides this weirdness inside the country/locality class, where it belongs.

Reply:
can `place_code` on Task actually just go away altogether?

Reply:
Yes, that makes sense to me.